### DISCUSSION TITLE
How do I map reddit's up and downvote to enable hints?

### DISCUSSION THREAD

**Discussion by nadir-ishiguro** [tridactyl/tridactyl#4718] [Q&A]
Hi,
I've been using the following binding for toggling reddit comments via hints for quite a while:

`bind ;c hint -Jc [class*="expand"],[class="togg"],[class="comment_folder"],[class="midcol"]`

I can't remember from where I got it but it works great, so I thought it should be relatively straightforward to extend it to include up and downvote buttons, but I haven't been able to.

I was hoping somebody here might be able to help me out.

**Comment by bovine3dom** [comment]
It's a long read but https://github.com/tridactyl/tridactyl/blob/master/doc/newsletters/tips-and-tricks/1-hint-css-selectors.md should give you all the tools you need to work it out :)
